customer/src/main/java/no/unit/nva/customer/CustomerDoiHandler.java

```diff
@@ -51,10 +80,33 @@ private boolean notEditor(RequestInfo requestInfo) {
 
     private boolean notInstAdmin(RequestInfo requestInfo) {
         return !requestInfo.userIsAuthorized(EDIT_OWN_INSTITUTION_USERS.toString());
-
     }
 
     private boolean notApplicationAdmin(RequestInfo requestInfo) {
         return !requestInfo.userIsApplicationAdmin();
     }
+
+    private UUID toUuid(URI customerId) {
+        return
+            UUID.fromString(
+                UriWrapper.fromUri(customerId).getLastPathElement()
+            );
+    }
+
+    // TODO add this code in task #NP-27814
```

Comment:
What about this?

Reply:
I went a little too fast, next qa is this change.